index.html

```diff
@@ -31,21 +31,45 @@
   </head>
   <body>
     <section id="abstract">
-      <p>This document aims to define WebViews</p>
+      <h2>Abstract</h2>
+      <p>This document aims to define what a <a>WebView</a> is.</p>
     </section>
     <section id="sotd">
+      <p>
+        This document is a work in progress. It is intended to provide clarity on the definition of a WebView, which has been a topic of discussion within the WebView Community Group.
+      </p>
     </section>
-    <section class="introductory">
-      <h2>Web Browser vs. WebView Definition</h2>
+    <section id="introduction">
+      <h2>Introduction</h2>
+      <p>
+        The WebView Community Group has identified a lack of clarity and consensus around the definition of a <a>WebView</a>. This has
+        led to confusion among developers and users alike, as the term is often used inconsistently across different platforms
+        and contexts.
+      </p>
       <p>
-        The WebView CG identified a need to come up with a definition to clearly identify a web browser vs. WebView. This was discussed via email
-        <a href="https://lists.w3.org/Archives/Public/public-webview/2025May/0000.html">here</a>.
+        <strong>Non-goal:</strong> The purpose of this document is to define what a WebView is. It does not attempt to categorize or define different types of WebViews (<a href="https://webview-cg.github.io/usage-and-challenges/#types-of-webviews">see the usages and challenges explainer</a> for that).
+
+        <aside class="note">
+          <p> TODO: Move types from other report here?</p>
```

Comment:
Should we make the WebView types we previously discussed in the usages and challenges docs part of this explainer?

Reply:
I decided to copy that here include it.